lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -0,0 +1,58 @@
+.reportback__permalink {
+  .header {
+    > .wrapper {
+      text-align: left;
+    }
+  }
+
+  .container {
+    &.-purple {
+      background: $purple;
+    }
+
+    .-white {
+      background: $white;
+    }
+
+    > .wrapper {
+      @include span(16);
```

Comment:
Oh gotcha. Yeah, I guess that stuff would need to be overridden then. Sorry, I hadn't seen the mobile view yet @_@ Just going by what I remembered for the desktop view :dancer: